submissions/ksalpern/Hooli-style-Popup/index.html

```diff
@@ -0,0 +1,269 @@
+                          <a href="#" class="popup__more-icon">
+                            <img
+                              src="./popup-icons/news.png"
+                              alt=""
+                              class="icon"
+                            />
+                            <span class="popup__more-name">News</span>
+                          </a>
+                        </li>
+
+                        <li class="popup__more-item">
+                          <a href="#" class="popup__more-icon">
+                            <img
+                              src="./popup-icons/photos.png"
+                              alt=""
+                              class="icon"
+                            />
+                            <span class="popup__more-name">Photos</span>
+                          </a>
+                        </li>
+
+                        <li class="popup__more-item">
+                          <a href="#" class="popup__more-icon">
+                            <img
+                              src="./popup-icons/promo.png"
+                              alt=""
+                              class="icon"
+                            />
+                            <span class="popup__more-name">Promo</span>
+                          </a>
+                        </li>
+
+                        <li class="popup__more-item">
+                          <a href="#" class="popup__more-icon">
+                            <img
+                              src="./popup-icons/search.png"
+                              alt=""
+                              class="icon"
+                            />
+                            <span class="popup__more-name">Search</span>
+                          </a>
+                        </li>
+
+                        <li class="popup__more-item">
+                          <a href="#" class="popup__more-icon">
+                            <img
+                              src="./popup-icons/tanslation.png"
+                              alt=""
+                              class="icon"
+                            />
+                            <span class="popup__more-name">Translate</span>
+                          </a>
+                        </li>
+
+                        <li class="popup__more-item">
+                          <a href="#" class="popup__more-icon">
+                            <img
+                              src="./popup-icons/videos.png"
+                              alt=""
+                              class="icon"
+                            />
+                            <span class="popup__more-name">Videos</span>
+                          </a>
+                        </li>
+
+                        <li class="popup__even-more">
+                          <a class="popup__more-even even-more" href="#">Even more from Kottans</a>
+                        </li>
```

Comment:
Add some margins to this element
![image](https://user-images.githubusercontent.com/16196199/182917487-22835a7b-444b-43c6-93d5-5bdccf7fdc1d.png)